ocs_ci/deployment/ibm.py

```diff
@@ -0,0 +1,227 @@
+        storage_classes = ocp.OCP(
+            kind='StorageClass')
+        try:
+            sclassess = [(sub['metadata']['name'], sub['provisioner'])
+                         for sub in storage_classes.get().get('items')]
+            scnoprovisioner = [sub[0] for sub in sclassess
+                               if sub[1] == 'kubernetes.io/no-provisioner']
+            scopenshiftprovisioner = [sub[0] for sub in sclassess
+                                      if not sub[1] == 'kubernetes.io/no-provisioner']
+        except (IndexError, CommandFailed):
+            logger.warning("Error")
+
+        storage_clusters = ocp.OCP(
+            kind='StorageCluster')
+        try:
+            scdevicesets = [sub['spec']['storageDeviceSets']
+                            for sub in storage_clusters.get().get('items')]
+            sclassindevicesets = \
+                [[sub1['dataPVCTemplate']['spec']['storageClassName']
+                    for sub1 in sub] for sub in scdevicesets]
+        except (IndexError, CommandFailed):
+            logger.warning("Error")
+
+        # sclassindevicesets is a list of lists. Each outer list is for
+        # a specific storage cluster. All storage classes in the inner
+        # list are same (same storage class used across all devicesets
+        # in a storage cluster. So pick only the first one.
+
+        # for each storage class, get LocalVolume (owner).
+
+        if len(sclassindevicesets) == 0:
+            logger.warning("No storage class in device sets. Quitting.")
+            return
+
+        # compare scnoprovisioner and sclassindevicesets
+        tmplist = []
+        for sclist in sclassindevicesets:
+            for sc in sclist:
+                if sc in scnoprovisioner:
+                    if sc not in tmplist:
+                        tmplist.append(sc)
+
+        scandlvlist = []
+        for sc in tmplist:
+            storage_class = ocp.OCP(
+                kind='StorageClass',
+                resource_name=sc
+            )
+            localvolumemeta = storage_class.get().get('metadata')
+            localvolume = localvolumemeta['labels']['local.storage.openshift.io/owner-name']
+            scandlvlist.append((sc[0], localvolume))
+
+        alldevicepaths = []
+        for lv in scandlvlist:
+            # Get localvolume and corresponding devices
+            localvolume = ocp.OCP(kind='LocalVolume',
+                                  namespace='local-storage',
+                                  resource_name=lv[1]).get().get('spec')
+            scdevices = localvolume['storageClassDevices']
+            devicepaths = [sub['devicePaths'] for sub in scdevices]
+            alldevicepaths.append(devicepaths[0][0])
+
+        # STEP 5
+        # delete storage cluster - all of them.
+        scdelete = 'oc delete -n openshift-storage storagecluster --all --wait=true'
+        logger.info(scdelete)
+        out = run_cmd(scdelete)
+        logger.info(out)
```

Comment:
`out` should already be being logged by `run_cmd`. I don't think we need to duplicate this here. Address on all other instances of this pattern here in `ibm.py`.